service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -44,14 +49,27 @@
     @MockBean
     private DocumentConversionService documentConversionService;
 
+    @MockBean
+    private FeatureToggleService featureToggleService;
+
+    private CaseManagementOrder cmo = buildCMO();
+    private DocumentReference convertedDocument;
+    private DocumentReference sealedDocument;
+
     ReviewCMOControllerAboutToSubmitTest() {
         super("review-cmo");
     }
 
+    @BeforeEach()
+    void setup() throws Exception {
+        convertedDocument = testDocumentReference();
+        sealedDocument = testDocumentReference();
+        given(documentConversionService.convertToPdf(cmo.getOrder())).willReturn(convertedDocument);
+        given(documentSealingService.sealDocument(convertedDocument)).willReturn(sealedDocument);
```

Comment:
To be honest I am not a fan of this change. Seal document is not something which is returned in every test I would possibly mock it in the specific test scenario. 

See this: https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html it's good read exploring that topic.

Reply:
We could move the above into a private method like givenTheDocumentsWereConvertedAndSealed(); and call this method in the tests that we need it for.. or @Jamesnelson117 you can revert to previous as in the givens mentioned in each test. up to yourself :)

Reply:
But yes Adam that's a good point! :)

Reply:
I'll just revert Toireasa. I think repetition in this case is fine.